projects/glitter-finance/index.js

```diff
@@ -1,9 +1,42 @@
 const ADDRESSES = require('../helper/coreAssets.json')
-//Solana Helpers
 const { sumTokensExport, } = require("../helper/sumTokens")
+
 //addresses
 const SOLANA_VAULT = "7xCU4nvqu3Nz3BBQckKzibp3kBav4xbkuqQ3WM9CBHdJ";
 const ALGO_VAULT = "R7VCOR74LCUIFH5WKCCMZOS7ADLSDBQJ42YURFPDT3VGYTVNBNG7AIYTCQ";
+const ETHEREUM_VAULT = "0xa234acbd98a917f6dda69298e0e7290380006cf1";
```

Comment:
what is the NOSEC token here?

on other evm chains, it is not a vault but bridged token?

Reply:
This is our vault for our bridged for glitter finance.  NOSEC gets bridged to xNOSEC on all the other chains.  THe vault on EThereum holds the bridged NOSEC.  We are in the process of listing the token with coingeko and cmc now.